src/main/java/pages/ShippingPage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ShippingPage {
+    protected WebDriver driver;
+    private By titlePageCheckout = By.name("Checkout");
+    private By address = By.className("shipping-address-items");
+    private By addressSelected = By.className("shipping-address-item selected-item");
+    private By detailsQuantity = By.className("details-qty");
+
+    public ShippingPage(WebDriver driver){
+        this.driver = driver;
+    }
+
+public String verifyTittlePage (){
+        return driver.findElement(titlePageCheckout).getText();
+}
+
+public String verifyAddress(){
+        return driver.findElement(address).getText();
+}
+public String verifyAddressSelected(){
+        driver.findElement(addressSelected).getCssValue("border-color");
+    return null;
+}
+public String verifyDetailQty(){
```

Comment:
What do you want to do with this method? the name tells me that you are verifying something, but in the body of the method I see that you get a text, perhaps it is "obtain the amount purchased" or something similar, verify it